openff/units/units.py

```diff
@@ -22,6 +22,15 @@
 DEFAULT_UNIT_REGISTRY = pint.UnitRegistry(get_defaults_path())
 """The default unit registry provided by OpenFF Units"""
 
+MD_UNITS = {
```

Comment:
```suggestion
_MD_UNITS = {
```
Maybe this should be "private"? I'd like to avoid all discussion about what units should be used in MD